src/components/Public/Record/Exporter/FieldPicker.vue

```diff
@@ -570,13 +587,15 @@ export default {
     },
 
     doExport (kind) {
+      this.processing = true
       this.$emit('export', {
         ext: kind,
         fields: this.fields.map(({ name }) => name),
         filters: this.makeFilters(this.filter),
         filterRaw: this.filter,
         timezone: this.forTimezone ? this.exportTimezone : undefined,
       })
+      this.processing = false
```

Comment:
This wont work since we just emit the event here, this means processing will be false as soon as its emitted and not actually processed. You need to set the processing flag to true/false in the parent component.